src/app/components/main/main.component.ts

```diff
@@ -37,6 +38,7 @@ export class MainComponent {
                 global: GlobalService,
                 electron: ElectronProxyService,
                 // DON'T REMOVE THIS PLEASE I KNOW IT DOESN'T HAVE ANY USAGES
```

Comment:
Move this comment to be above `js: JavascriptEvalService` because that's what it's referring to.

Also, change `DON'T REMOVE THIS` to `DON'T REMOVE THE JavascriptEvalService` so it's more obvious what the comment is referring to.